app/views/schools/rejected_requests/index.html.erb

```diff
@@ -0,0 +1,74 @@
+<%
+  self.breadcrumbs = {
+    @current_school.name => schools_dashboard_path,
+    'Rejected requests' => nil
+  }
+%>
+
+<div id="withdrawn-requests-index" class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= page_heading 'Rejected requests' %>
+
+    <% if @requests.any? %>
+      <div class="pagination-info higher">
+        <div class="pagination-slice">
+          <%= page_entries_info @requests %>
+        </div>
+        <%= paginate @requests %>
+      </div>
+
+      <table id="rejected-requests" class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th class="govuk-table__header" scope="col">Name</th>
+            <th class="govuk-table__header" scope="col">Subject</th>
+            <th class="govuk-table__header" scope="col">Date requested</th>
+            <th class="govuk-table__header" scope="col">Reason for rejection</th>
+            <th class="govuk-table__header"></th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <%- @requests.each do |rejected_request| -%>
+            <tr class="rejected-request govuk-table__row" data-placement-request-id="<%= rejected_request.id %>">
+              <td class="name govuk-table__cell">
+                <%= rejected_request.candidate_name %>
+              </td>
+              <td class="subject govuk-table__cell">
+                <%- if rejected_request&.placement_date&.subject_specific -%>
+                  <%= rejected_request.placement_date.subjects.map(&:name).to_sentence %>
```

Comment:
I don't think this is the way to get at the subject for a subject specific placement request as it lists all the subjects for the date rather than the applied for subject. Subject specific dates pr adds a subject to placement requests so probably want to use that `if rejected_request.subject ...`